si/orange3.jaml

```diff
@@ -4398,70 +4398,122 @@ widgets/data/owconcatenate.py:
     iris: false
     zoo: false
 widgets/data/owcontinuize.py:
+    Use preset: Uporabi prednastavljeno
+    preset: prednastavljeno
```

Comment:
Mogoče kar "privzeto"? Res bi bilo to v angleščini default, ampak ... ne vem, razmisliva.